graders/c/cgrader/cgrader.py

```diff
@@ -49,6 +50,12 @@
 ASAN_FLAGS = ("-fsanitize=address", "-static-libasan", "-g", "-O0")
 
 
+class OutputMatchingOption(enum.Enum):
```

Comment:
```suggestion
import typing as t

OutputMatchingOption = t.Literal["all", "partial", "any"]
```

I think this is closer to what you want. Don't over-worry about it since these annotations aren't read at runtime anyway.

Reply:
Thanks, this is useful. Done in fa6c7d8.